framework/py/flwr/common/inflatable_grpc_utils.py

```diff
@@ -48,61 +45,100 @@
 }
 
 
+class ObjectUnavailableError(Exception):
+    """Exception raised when an object has been pre-registered but is not yet
+    available."""
+
+    def __init__(self, object_id: str):
+        super().__init__(f"Object with ID '{object_id}' is not yet available.")
+        self.object_id = object_id
```

Comment:
Why do we need to keep it around ? (same goes for the `ObjectIdNotPreregisteredError` below)